src/Globe/TileMesh.js

```diff
@@ -195,37 +196,6 @@ TileMesh.prototype.setSelected = function(select) {
     this.materials[RendererConstant.FINAL].setSelected(select);
 };
 
-TileMesh.prototype.parseBufferElevation = function(image, minMax, pitScale) {
```

Comment:
This function computes the bounding box's tile.
This bbox is used to compute bounding sphere.
This bounding sphere is used to compute the screen space error. 
If this function is removed the screen space error is too big, and tiles are subdivided or displayed unnecessarily.
we encountered performance problems without this optimization. 
Furthermore the bouding spheres will be used for collision.